core/vm/opcodes.go

```diff
@@ -117,6 +117,8 @@ const (
 	GAS      OpCode = 0x5a
 	JUMPDEST OpCode = 0x5b
 	PUSH0    OpCode = 0x5f
+	TLOAD    OpCode = 0xb3
```

Comment:
These should probably be in a latter range, this range is the `0x50 -range`. So somewhere further down